pkg/cloudprovider/providers/rackspace/rackspace.go

```diff
@@ -595,23 +571,23 @@ func (rs *Rackspace) DetachDisk(instanceID string, partialDiskId string) error {
 		return err
 	}
 
-	if len(disk.Attachments) > 1 {
+	if len(volume.Attachments) > 1 {
 		// Rackspace does not support "multiattach", this is a sanity check.
-		errmsg := fmt.Sprintf("Volume %s is attached to multiple instances, which is not supported by this provider.", disk.ID)
+		errmsg := fmt.Sprintf("Volume %s is attached to multiple instances, which is not supported by this provider.", volume.ID)
 		return errors.New(errmsg)
 	}
 
-	if len(disk.Attachments) > 0 && instanceID == disk.Attachments[0]["server_id"] {
+	if len(volume.Attachments) > 0 && instanceID == volume.Attachments[0]["server_id"] {
 		// This is a blocking call and effects kubelet's performance directly.
 		// We should consider kicking it out into a separate routine, if it is bad.
-		err = volumeattach.Delete(compute, instanceID, disk.ID).ExtractErr()
+		err = volumeattach.Delete(compute, instanceID, volume.ID).ExtractErr()
 		if err != nil {
-			glog.Errorf("Failed to delete volume %s from compute %s attached %v", disk.ID, instanceID, err)
+			glog.Errorf("Failed to delete volume %s from compute %s attached %v", volume.ID, instanceID, err)
 			return err
 		}
-		glog.V(2).Infof("Successfully detached volume: %s from compute: %s", disk.ID, instanceID)
+		glog.V(2).Infof("Successfully detached volume: %s from compute: %s", volume.ID, instanceID)
 	} else {
-		errMsg := fmt.Sprintf("Disk: %s has no attachments or is not attached to compute: %s", disk.Name, instanceID)
+		errMsg := fmt.Sprintf("Disk: %s has no attachments or is not attached to compute: %s", volume.Name, instanceID)
```

Comment:
Probably not, but it was already there. This PR was just concerned with clarifying how UUIDs are used in function signatures.